### ISSUE TITLE
bloodhound AJAX options are missing - POST not possible

### ISSUE THREAD

**Issue by onemenny** [twitter/typeahead.js#1181]
v0.11.0 omitted the ajax options for prefetch and remote, it seems that the 'GET' is now hardcoded. 
how can a POST operation be preformed?

**Comment by jharding** [comment]
You'll want to use the [`prepare` option](https://github.com/twitter/typeahead.js/blob/master/doc/bloodhound.md#remote). This change was made to make things more generic so the transport mechanism could be swapped out. Here's an example of how you'd do it:

```js
function prepare(query, settings) {
  settings.type = 'POST';
  return settings;
}

var b = new Bloodhound({
  remote: {
    // ...
    prepare: prepare
  }
});
```

**Comment by onemenny** [comment]
but that option is missing from prefetch...

**Comment by jharding** [comment]
That it is... woops. I'll get that fixed for v0.11.1 (will probably be released tomorrow). Thanks for the report.

**Comment by onemenny** [comment]
thanks, will be waiting for that

**Comment by jharding** [comment]
PR is up, see #1184. I'll get it merged into `integration-0.11.1` tomorrow before I do the release.
